docs/integrations/bitbucket/discovery.md

```diff
@@ -0,0 +1,43 @@
+---
+id: discovery
+title: Bitbucket Server Discovery
```

Comment:
So, a question about naming - the word "server" stands out to me here. Firstly it sounds a bit like its purpose is to discover servers :) I see that it doesn't support Bitbucket cloud, but is that cause enough to affect the naming? Is it a feature that's impossible to implement for the cloud product or is it just on todo? Could it be implemented to only throw an exception if you try to use wildcards in the project part, on cloud, if that particular piece isn't available?

Suggest maybe renaming this (and all other places) to remove the word "server", and perhaps call it the "Bitbucket Discovery Processor" plain and simple, so it's aligned with the class name and with the GitHub Discovery Processor. Of course we'd want to clearly document (and assert in code) that this isn't to be used for the cloud product at the moment

Reply:
I have removed all occurrences of Server and it is stated in the documentation that it currently does not support Bitbucket Cloud accounts. There is also a validation step in the processor to make sure that it does not try to read from bitbucket.org

https://github.com/backstage/backstage/blob/13abbc15da7743005cfe6c9123499af73eddb728/plugins/catalog-backend/src/ingestion/processors/BitbucketDiscoveryProcessor.ts#L65-L67